packages/kit/src/cli.js

```diff
@@ -70,8 +70,9 @@ prog
 	.describe('Start a development server')
 	.option('-p, --port', 'Port', 3000)
 	.option('-h, --host', 'Host (only use this on trusted networks)', 'localhost')
+	.option('--https', 'use HTTPS', false)
```

Comment:
Do we want to say "with a self signed cert" here or anything like that?

Reply:
done